modules/nf-core/md5sum/tests/main.nf.test

```diff
@@ -0,0 +1,73 @@
+nextflow_process {
+
+    name "Test Process MD5SUM"
+    script "../main.nf"
+    process "MD5SUM"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "md5sum"
+
+    test("md5sum on hello.txt") {
+
+        when {
+            process {
+                """
+                input[0] = Channel.of([
+                    [ id: 'hello' ],
+                    [ file(params.modules_testdata_base_path + 'generic/txt/hello.txt', checkIfExists: true) ]
+                ])
+                """
+            }
+        }
+
+        then {
+            with(process.out) {
```

Comment:
assertAll(
                { assert process.success },
                { assert snapshot(process.out).match() }
            )
            
Should be working just fine for this module, no? (same for next test hopefully)
Also need a versions snapshot, and these two lines (if working properly) will cover that.

Reply:
You're right, that's much simpler! This was my first attempt to use nf-test so I may have gotten ahead of myself...